### ISSUE TITLE
CMD/CTRL + K save prompts and quickly load in instructions

### ISSUE THREAD

**Issue by pjsandwich** [getcursor/cursor#1199]
**Describe the solution you'd like**
1. When using cmd/ctrl + k, you should be able to reference saved instructions.
2. The instructions could either be saved in editor settings or the project, but preferably the editor settings.

**Additional context**
I typically use the cmd/ctrl + k to create styles for a file or to edit code using simple instructions. I typically use the same prompts like “Create a stylesheet for this component and place the styles at the bottom of the file.” and it works wonderfully, but I’m wondering if we could save prompts somewhere and possibly reference them so that it could save some time and reduce small inconsistencies in applying instructions.

[More informal discussion post](https://forum.cursor.sh/t/saving-prompts-for-cmd-ctrl-k-instructions/2546?u=pjsandwich)

**Comment by TJtangjun** [comment]
agree, that I can save my often-used command to run it quickly, not to ask ai over and over again with the same words.

**Comment by truell20** [comment]
Does the "Rules for AI" in the "More" tab next to chat solve this use-case? Or up arrowing through past prompts? If not, curious to hear what you'd want out of a good experience here.

**Comment by TJtangjun** [comment]
> Does the "Rules for AI" in the "More" tab next to chat solve this use-case? Or up arrowing through past prompts? If not, curious to hear what you'd want out of a good experience here.

It's embarrassing that as a long-time user I didn't realize that pressing the arrow keys would show me the history of inputs, there was no hint that told me I could do that. Since this feature allows me to see the history, I think it basically fulfills that need.

---

I mean, consider adding something like `//command`, for example, if I type `//simplify`, it will simplify the current code for me. Of course, these commands can be customized. 

The reason I use two `/` is that I don't want to conflict with Cursor own functions.

---

If my suggestion is too difficult, use **history** is ok too. Just a little advice.
